### PR TITLE
apb tests - disable openshift tests

### PR DESCRIPTION
**Pull request by spolti**
disable test on openshift since the old registry is not supported anymore
and the new requires auth.

Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,37 +5,33 @@ sevices:
...
 
+  # workaround for bypass odcs auth
+  - |
+    sed -i "s/  content_sets_file: content_sets.yml//g" ${PWD}/image.yaml
   # Install and run cekit
-  - virtualenv ~/cekit
-  - source ~/cekit/bin/activate
-  - pip install -q -U cekit
-  - cekit build --overrides-file dev-overrides.yaml
+  - pip install --pre docker docker-squash cekit odcs
```
Comment (errantepiphany) [inline .travis.yml]: Would it be safer to specify a particular version of cekit here? For example, I do this:
pip3 install -U 'cekit>=3.2.0,<3.3.0

File: .travis.yml [hunk 2]
```diff
@@ -5,37 +5,33 @@ sevices:
...
+#  allow_failures:
+#    - env: OPENSHIFT_VERSION=v3.10
 
 script:
   # Configure test values
   - export apb_name='rhpam-apb'
   - export dockerfile='target/image/Dockerfile'
-  - export resources='https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/7.3.x/rhpam73-image-streams.yaml'
-  - export resources_namespace='openshift'
+  - export image_stream='https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/7.4.1.GA/rhpam74-image-streams.yaml'
```
Comment (errantepiphany) [inline .travis.yml]: Should a tag (7.4.1.GA) really be used here? Or should it be the branch for which this file lives? (So, "7.5.x" or "master"?)

File: apb-test.sh [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        wget https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
+        tar -xzvf openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz --strip-components=1 openshift-origin-client-tools-v3.9.0-191fece-linux-64bit/oc
+    fi
+    sudo cp oc /usr/local/bin/
+    sudo cp /usr/local/bin/oc /usr/local/bin/kubectl
+    oc cluster up \
+        --skip-registry-check=true \
+        --routing-suffix=172.17.0.1.nip.io \
+        --public-hostname=172.17.0.1 \
+        --enable=service-catalog,template-service-broker,router,registry,web-console,persistent-volumes,sample-templates,rhel-imagestreams
```
Comment (errantepiphany) [inline apb-test.sh]: Why do you need the web-console or sample-templates?

File: .travis.yml (hunk 1 above)
Comment (spolti) [reply .travis.yml]: It really does not matter.

File: .travis.yml (hunk 2 above)
Comment (spolti) [reply .travis.yml]: it is not used anymore, maybe comment it out can be better.

File: apb-test.sh (hunk 3 above)
Comment (spolti) [reply apb-test.sh]: it is not called, I just keept it as on the original script.
the lines which call those functions are commented.

File: .travis.yml (hunk 1 above)
Comment (spolti) [reply .travis.yml]: but, it is a good idea since new cekit versions can break stuff.

File: .travis.yml (hunk 2 above)
Comment (spolti) [reply .travis.yml]: line commented out.

**Pull request merged**